infrastructure/instances.tf

```diff
@@ -253,10 +233,10 @@ data "template_file" "nomad_client_script_smusher" {
 resource "aws_instance" "nomad_client_1" {
   ami = "${data.aws_ami.ubuntu.id}"
   instance_type = "t2.xlarge"
-  availability_zone = "${var.region}b"
+  availability_zone = "${var.region}a"
   vpc_security_group_ids = ["${aws_security_group.data_refinery_worker.id}"]
```

Comment:
Again, using a generic `worker` SG. Should really be per task type, and need a further dive into how Nomad allocates jobs on difference instance types and in different SGs.